### DISCUSSION TITLE
One of my pages refuses to have CSS applied to it.

### DISCUSSION THREAD

**Discussion by BestofTheBread** [community/community#126289] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Repo: [https://github.com/BestofTheBread/Newboes-Homework-Help](https://github.com/BestofTheBread/Newboes-Homework-Help)
Both of my html files are linked to the same file, however one of them does not have the CSS applied to it. Is there any way I can fix this?

**Comment by LautaroNores** [comment]
It sounds like you're experiencing an issue where CSS is not being applied to one of your HTML files, despite both HTML files being linked to the same CSS file. This can happen due to various reasons. Here are a few troubleshooting steps you can try:

Check CSS File Path:

Ensure that the CSS file path specified in the HTML file that's not applying CSS is correct and points to the actual location of the CSS file.
Inspect Browser Console:

Open the developer tools in your browser (usually by pressing F12 or right-clicking and selecting "Inspect").
Navigate to the "Console" tab and look for any error messages related to loading or parsing the CSS file.
Verify CSS Link Tag:

Double-check the <link> tag in your HTML file that references the CSS file. Ensure that the href attribute points to the correct location and that there are no typos or errors in the tag.
Check CSS Syntax:

Open the CSS file and verify that the syntax is correct. Any errors in the CSS file could prevent it from being applied properly.
Clear Browser Cache:

Sometimes, cached versions of CSS files can cause issues. Try clearing your browser cache and reloading the page to ensure that the latest version of the CSS file is being used.
Test CSS Directly:

To isolate the issue, try opening the CSS file directly in your browser to see if it loads correctly. If it doesn't, there may be an issue with the CSS file itself.
Inspect HTML Structure:

Check the HTML structure of the page where CSS is not being applied. Ensure that there are no conflicting styles or CSS overrides that might prevent the styles from being applied as expected.
If you've tried these steps and are still experiencing issues, it could be helpful to provide more specific details or share snippets of your HTML and CSS code for further analysis.

Hope this helps!

**Reply by ebndev** [reply]
Hi @LautaroNores, thanks for being a part of the GitHub Community!

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ebndev** [comment]
Hi @BestofTheBread! Are you still having this issue? I'm seeing the CSS applied to both html files.

**Reply by BestofTheBread** [reply]
I fixed it some time ago it turns out that I was mixing it up between styles.css and style.css
